proposals/0400-init-accessors.md

```diff
@@ -358,7 +384,71 @@ init(x: Int, y: Int) {
 }
 ```
 
-Use cases for `init` accessors that provide a projection of a stored property as various units through several computed properties don't have a single preferred unit from which to initialize. Most likely, these use cases want a different member-wise initializer for each unit that you can initialize from. If a type contains several computed properties with `init` accessors that initialize the same stored property, a member-wise initializer will not be synthesized.
+Therefore, the compiler will order the initializations in the synthesized memberwise initializer to respect the `accesses` clauses:
+
+```swift
+init(x: Int, y: Int) {
+  self.y = y
+  self.x = x
+}
+```
+
+The initial review of this proposal suppressed the memberwise initializer in such cases, based on a concern that out-of-order initialization would cause surprises. However, given the fact that the fields are initialized independently (or have `accessses` relationships that defiee their relative ordering), and that side effects here are limited to those of the `init` accessors themselves, one has to introduce global side effects during initialization to observe any difference.
```

Comment:
```suggestion
The initial review of this proposal suppressed the memberwise initializer in such cases, based on a concern that out-of-order initialization would cause surprises. However, given the fact that the fields are initialized independently (or have `accessses` relationships that define their relative ordering), and that side effects here are limited to those of the `init` accessors themselves, one has to introduce global side effects during initialization to observe any difference.
```